import/src/main/java/com/google/cloud/healthcare/imaging/dicomadapter/ImportAdapter.java

```diff
@@ -164,10 +164,11 @@ private static DicomRedactor configureRedactor(Flags flags) throws IOException{
     DestinationsConfig conf = new DestinationsConfig(destinationJsonInline, destinationsJsonPath);
     Map<DestinationFilter, IDicomWebClient> result = new LinkedHashMap<>();
     for (String filterString : conf.getMap().keySet()) {
+      String filterPath = StringUtil.trim(conf.getMap().get(filterString));
```

Comment:
Can we validate the filter paths at the start of execution as well?

Reply:
Validation already happens in DicomWebClien/DicomWebClientJetty constructors, invoked as part of startup.